### ISSUE TITLE
Fine tuning pretrained models but it turns out the pretrained model does not have encoder part.

### ISSUE THREAD

**Issue by YunyaGaoTree** [SarahwXU/HiSup#14]
Dear Sarah,

I would like to kindly ask how to use pretrained model (e.g. hrnet-w48-v2 backbone) as the following screenshot shows.
![issue1](https://github.com/SarahwXU/HiSup/assets/101531630/01137e26-2d8b-4eea-8e56-ec3647e5c6be)

What I have done is loading this pretrained model and then use the train function to start fine-tuning.
As this screenshot shows:
![issue0](https://github.com/SarahwXU/HiSup/assets/101531630/7d157d2d-59b6-4f60-9a57-f0d6c0cfc6f4)

However, it always has an error 'BuildingDetector' object has no attribute 'encoder' as the following shows:
![issue](https://github.com/SarahwXU/HiSup/assets/101531630/f416d0a8-6b8f-4f1c-ae65-6f0f066bf2af)

I have used the pretrained model for inference. It works quite well. That is why I would like to fine-tune it to improve its performance further. 

Hope to get answers from you. 
Thanks so much for providing this great work and look forward to your reply.

**Comment by XJKunnn** [comment]
Hi,

We have only set the encoder module for training mode as shown in the code below. https://github.com/SarahwXU/HiSup/blob/69e0cb3c7039bd038139ae0b95ea4ef03b2e5162/hisup/detector.py#L69-L71 
so if you want to finetune from the pretrained models, you need to disable testing mode. https://github.com/SarahwXU/HiSup/blob/69e0cb3c7039bd038139ae0b95ea4ef03b2e5162/hisup/detector.py#L234
